tests/strangelove/suite.go

```diff
@@ -190,11 +190,11 @@ func (s *Suite) SetupChain(chain *SuiteChain) {
 
 	chain.Note = s.Instantiate(cc, user, noteId, NoteInstantiate{})
 	chain.Voice = s.Instantiate(cc, user, voiceId, VoiceInstantiate{
-		ProxyCodeId: uint64(proxyUint),
-		BlockMaxGas: 100_000_000,
+		ProxyCodeId:     uint64(proxyUint),
+		BlockMaxGas:     100_000_000,
+		ContractAddrLen: 32,
```

Comment:
Running ict here does not work due to v4 not supporting genesis sub commands (SDK v47 moved add-genesis-account -> genesis add-genesis-account)

Juno "v15.0.0" should fix this in the NewSuite setup instead of "latest"

```
				Name:          "juno",
				ChainName:     "juno2",
				Version:       "v15.0.0",
```

Reply:
it's not working and they're not in use right now :(